src/Transport/HttpAdapter.php

```diff
@@ -30,7 +30,7 @@ class HttpAdapter extends AbstractTransport
     /**
      * Construct transport.
      */
-    public function __construct(?Connection $connection = null, HttpAdapterInterface $httpAdapter)
+    public function __construct(?Connection $connection, HttpAdapterInterface $httpAdapter)
```

Comment:
This is odd indeed. I wonder if the adapter should be the first param. But I know, it would be a breaking change.

Reply:
Yes, I had the same thought.